test/bench/bench.cpp

```diff
@@ -112,13 +113,14 @@ void validate_benchmark_case(benchmark::State& state, fizzy::test::WasmEngine& e
 
     if (benchmark_case.expected_result)
     {
+        std::string expected = std::to_string(*benchmark_case.expected_result);
         if (!result.value)
-            return state.SkipWithError("Missing result value");
+            return state.SkipWithError(("Missing result value (expected: " + expected.data() + ")"));
         else if (*result.value != *benchmark_case.expected_result)
-            return state.SkipWithError("Incorrect result");
+            return state.SkipWithError("Incorrect result value");
     }
     else if (result.value)
-        return state.SkipWithError("Unexpected result");
+            return state.SkipWithError(("Unexpected result value: " + std::to_string(*result.value)).data());
```

Comment:
When I was adding the benchmark it was always frustrating getting the right values out. Not sure what is the nicest way to add error messages here.